R/vector_parameters.R

```diff
@@ -46,6 +46,20 @@ fun_params <- list(
   mum = .112
 )
 
+kol_params <- list(
```

Comment:
You've missed out the `@export` tag. Here's some [documentation on it](https://kbroman.org/pkg_primer/pages/docs.html)
```suggestion
#' @export
kol_params <- list(
```